app/models/early_allocation.rb

```diff
@@ -95,7 +95,7 @@ def validate_stage2
             inclusion: { in: [true, false], allow_nil: false },
             if: -> { recording_community_decision }
 
-  def eligible?
+  def auto_eligible?
```

Comment:
What's the correct name for this? I'm not convinced it should be renamed.

I'm working on MO-11 which exclusively uses the term "eligible" – even in the UI to users. It **does not** call them "auto eligible".

Reply:
This means auto_eligible - hence the rename. I found it confusing when working on the allocation history story (MO-20) as a case can be eligible but say eligible? false...!